.gitignore

```diff
@@ -10,3 +10,4 @@ sitesearch-volume-data
 kubernetes/secrets.yaml
 /.vscode
 /volumes
+/.idea
```

Comment:
Perhaps add this to a global gitignore? https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

I don't mind at all though.

Reply:
I didn't know about that. I've added it, thank you!

I'll still keep `.idea` here, though, since `.vscode` is also there. Code editor equality!